cognite/neat/rules/_analysis/_information_rules.py

```diff
@@ -263,12 +337,13 @@ def as_class_dict(self) -> dict[str, InformationClass]:
             class_dict[definition.class_.suffix] = definition
         return class_dict
 
-    def subset_rules(self, desired_classes: set[ClassEntity]) -> InformationRules:
+    def subset_rules(self, desired_classes: set[ClassEntity], use_reference_rules: bool = False) -> InformationRules:
```

Comment:
Is this necessary? If you want to use the reference rules shouldn't you do:
```python
InformationArchitectRulesAnalysis(rules.reference).subsest_rules(desired_classes)
```
?